ci/kind/kind-setup.sh

```diff
@@ -665,6 +666,21 @@ if version_lt "$kind_version" "0.12.0" && [[ "$KUBE_PROXY_MODE" == "none" ]]; th
     exit 1
 fi
 
+# Build Kind node image with the specified K8s version if it doesn't exist in local and remote registry neither
+kind_node_image="kindest/node:${K8S_VERSION}"
+if docker pull "${kind_node_image}" > /dev/null 2>&1; then
+    echo "=== Using prebuild Kind node image with the K8s version ${K8S_VERSION} ==="
+else
+    echo "=== Building Kind node image with the K8s version ${K8S_VERSION} ==="
+    git clone https://github.com/kubernetes/kubernetes.git ${THIS_DIR}/kubernetes
```

Comment:
The script could be used by developers, we should use a temperary directory for the kubernetes repo and remove it after building, otherwise the script is not rerunable and significantly affect disk usage of developers.

Or maybe we could move this to conformance.yml, and just assume the image is available in this script.

Reply:
Moved this part to `conformance.yml`.